src/main/resources/static/common/js/common.js

```diff
@@ -85,6 +85,7 @@ function initHeaderController(app){
         }).then(function successCallback(response) {
             var preferenceData = response.data.data;
             var industry = preferenceData.industryType;
+            $rootScope.mqType = preferenceData.mqType;
```

Comment:
I think this variable can be removed and you can use only "$rootScope.displayKafkaError" with current condition logic to set it.